bin/foreman-tasks

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+foreman_root = '/usr/share/foreman'
```

Comment:
This shouldn't be run on your local checkout (just like you're not running bin/foreman-tasks from the plugin folder). This is moved to /usr/bin , then the systemd service calls it (https://github.com/theforeman/foreman-packaging/pull/1549/files#diff-a46acbb74d582e9d3a8ee262a72b237cR11). 

On local setups as yours, you should use https://github.com/theforeman/foreman-tasks/#production-mode if you want to have the executor separate from the Rails process.